lib/generators/sufia/templates/catalog_controller.rb

```diff
@@ -80,6 +80,8 @@ def self.modified_field
     config.add_index_field solr_name("resource_type", :stored_searchable), label: "Resource Type", link_to_search: solr_name("resource_type", :facetable)
     config.add_index_field solr_name("file_format", :stored_searchable), label: "File Format", link_to_search: solr_name("file_format", :facetable)
     config.add_index_field solr_name("identifier", :stored_searchable), label: "Identifier", helper_method: :index_field_link, field_name: 'identifier'
+    config.add_index_field solr_name("embargo_release_date", :stored_sortable, type: :date), label: "Embargo release date", helper_method: :human_readable_date
```

Comment:
@jcoyne @cbeer do you know why this might pass a Solr doc to `#human_readable_date` where `:value` is an array? This is a single-valued field.